CHANGELOG.md

```diff
@@ -2,9 +2,39 @@
 All notable changes to this project will be documented in this file.
 This project aims to adhere to [Semantic Versioning](http://semver.org/).
 
-## [3.3.2] -
+## [3.4.0] -
+### Added
+ - [Updated Prune-Depth Documentation. Add Unit Tests for more limit values](https://github.com/joyent/java-manta/pull/484)
 ### Fixed
- - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486) 
+ - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486)
```

Comment:
Shouldn't this line not be there?

Reply:
Didn't get you ?

Reply:
This isn't the same file that I commented on. Perhaps github had a consistency issue.